src/security/templatetags/securitytags.py

```diff
@@ -87,3 +90,26 @@ def is_repository_manager(context):
 def is_preprint_editor(context):
     request = context['request']
     return request.user.is_preprint_editor(request)
+
+
+@register.filter
+def se_can_see_pii(value, article):
+    request = GlobalRequestMiddleware.get_current_request()
+
+    if logic.can_see_pii(request, article):
+        return value
+    else:
+        return 'Value Anonymised'
```

Comment:
I don't think the word "value" is quite as clear outside of computer science as we'd want it to be for this user-facing text.

How about "[Anonymised data]" with the brackets to show it is being put in place of something else?

Reply:
+1

And flag it for translation